pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -340,6 +342,7 @@ func (exp *flowExporter) sendFlowRecords() error {
 				return err
 			}
 			exp.numDataSetsSent = exp.numDataSetsSent + 1
+			klog.V(4).InfoS("Deny connection sent successfully", "flowKey", connKey, "record", conn)
```

Comment:
s/record/connection/?

Reply:
Yes, that's correct. We need to keep it consistent with L334 but missed it here. Rephrased the log for correctness.